bitnami/wordpress/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.customPostInitScripts }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "wordpress.fullname" . }}-custominit-configmap
+data:
+{{ toYaml .Values.customPostInitScripts | indent 2}}
```

Comment:
Could you use here the `wordpress.tplValue` function instead of `toYaml` and `nindent` instead of `indent` please?

Reply:
Done